### PR TITLE
Change cli flag

### PR DESCRIPTION
**Pull request by dongrie**
tokenAddress -> denom

### CODE REVIEW

File: cmd/balance.go [hunk 1]
```diff
@@ -28,18 +28,18 @@ func balanceCmd() *cobra.Command {
...
-	cmd.MarkFlagRequired("token-address")
+	cmd.MarkFlagRequired("denom")
 
 	return cmd
 }
 
-func balanceOf(rpcAddress, ics20BankAddress, walletAddress, tokenAddress string) (*big.Int, error) {
-	baseDenom := strings.ToLower(tokenAddress)
+func balanceOf(rpcAddress, ics20BankAddress, walletAddress, denom string) (*big.Int, error) {
+	baseDenom := strings.ToLower(denom)
```
Comment (siburu) [inline cmd/balance.go:42]: Basically OK. BTW why do you use the name `baseDenom` instead of just `denom`?

File: cmd/transfer.go [hunk 2]
```diff
@@ -93,7 +92,7 @@ func Transfer(ctx context.Context, rpcAddress string, mnemonic, ics20BankAddress
 	}
 	log.Printf("2. deposit success (TxHash: %s)\n", tx.Hash().Hex())
 
-	baseDenom := strings.ToLower(tokenAddress)
+	baseDenom := strings.ToLower(denom)
```
Comment (siburu) [inline cmd/transfer.go:95]: Basically OK. BTW why do you use the name `baseDenom` instead of just `denom`?

File: cmd/balance.go (hunk 1 above)
Comment (siburu) [reply cmd/balance.go:42]: Is `strings.ToLower` needed?

File: cmd/transfer.go (hunk 2 above)
Comment (siburu) [reply cmd/transfer.go:95]: Is `strings.ToLower` needed?

Comment (siburu) [reply cmd/transfer.go:95]: It's not necessarily correct to apply `strings.ToLower` for port-id and channel-id.

File: cmd/balance.go (hunk 1 above)
Comment (siburu) [reply cmd/balance.go:42]: It's not necessarily correct to apply `strings.ToLower` for port-id and channel-id.

Comment (dongrie) [reply cmd/balance.go]: Fix variable name to `denom`
https://github.com/datachainlab/ethereum-ics20-cli/pull/4/commits/882f5f0dd56ebc20b7c908f7b1b2e6ebb086e51b

File: cmd/transfer.go (hunk 2 above)
Comment (dongrie) [reply cmd/transfer.go]: I removed the toLower and tried, Could not get the correct value.
スクリーンショット 2023-05-20 14 27 52

I tried to find the cause. ICS20Bank handles the following
https://github.com/hyperledger-labs/yui-ibc-solidity/blob/main/contracts/apps/20-transfer/ICS20Bank.sol#L79

`Strings.toHexString(tokenContract)` tested, return lower-case value
スクリーンショット 2023-05-23 8 33 41

Comment (siburu) [reply cmd/transfer.go]: I mean that if it's necessary to lower cases it should be done outside this command.

Comment (dongrie) [reply cmd/transfer.go]: https://github.com/datachainlab/ethereum-ics20-cli/pull/4/commits/5a4ebbfc45df0541caf27c3c8cf7aceda52bf47b
Fixed

**Review by siburu — APPROVED** [review]
LGTM. Thank you!

**Pull request merged**